flask_api.py

```diff
@@ -45,9 +45,16 @@
 app = Flask(__name__)
 CORS(app)
 
-# TODO: Initialize these somewhere else
-db = NimbusMySQLAlchemy()
-nimbus = Nimbus(db)
+db = None
```

Comment:
went with the approach that @cameron-toy proposed, I agree that this singleton-esque way to handle it is pretty good